### ISSUE TITLE
Uninstall Instruction

### ISSUE THREAD

**Issue by UprootedSeeker** [sailro/EscapeFromTarkov-Trainer#43]
I can't find uninstall instructions after using universal installer to install initially.

**Comment by sailro** [comment]
Indeed, perhaps I should add a command switch like '-uninstall' to remove everything automatically as well.

So manual steps are:
- remove the file `EscapeFromTarkov_Data\Managed\NLog.EFT.Trainer.dll`
- remove the file `EscapeFromTarkov_Data\outline`
- edit the file `EscapeFromTarkov_Data\Managed\NLog.dll.nlog`, remove the line `<target name="EFTTarget" xsi:type="EFTTarget" />`
